sdk/digitaltwins/tests.yml

```diff
@@ -0,0 +1,11 @@
+trigger: none
+
+extends:
+  template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: digitaltwins
+    Location: westus2
+    SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
+    EnvVars:
+      # Runs live tests.
+      AZURE_IOT_TEST_MODE: Live
```

Comment:
This may need to be `LIVE`, can't remember if what we check is case sensitive or not.

Reply:
We have it lower cased in the .NET repo and it works as expected.